2022/09/15 11:29:03 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.6:9000", host: "localhost"
2022/09/16 08:45:45 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.6:9000", host: "172.17.0.1"
2022/09/16 08:45:49 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.6:9000", host: "localhost"
2022/09/19 08:42:50 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.6:9000", host: "localhost"
2022/09/19 11:10:13 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 07:10:46 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:11:48 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcached HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:11:48 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcached HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:16:20 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcached HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:16:20 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcached HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:18:05 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:18:05 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:18:09 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:18:09 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:19:47 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:19:47 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:19:51 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:19:51 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:24:51 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:24:51 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:24:54 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:24:54 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:30:59 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:30:59 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:31:03 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:31:03 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:31:12 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcached HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:31:12 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcached HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:31:18 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:31:18 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:32:27 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:32:27 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:32:36 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:32:36 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:33:18 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:33:19 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:36:32 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:36:32 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:36:58 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:36:58 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:37:15 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:37:15 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:38:25 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:38:25 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:38:37 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:38:37 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:38:41 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:38:42 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:40:26 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 07:40:26 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:14:45 [error] 8#8: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:14:45 [error] 8#8: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:14:49 [error] 8#8: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:14:49 [error] 8#8: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:15:19 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:15:19 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:15:22 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:15:22 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:15:28 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:15:32 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:15:43 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:15:43 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:17:13 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:17:14 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:19:38 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcashe HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:19:53 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:19:53 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:21:51 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:21:52 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "172.17.0.1"
2022/09/22 08:22:08 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:22:08 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:24:29 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:24:29 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:24:36 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:24:56 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:25:35 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:25:57 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:26:22 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:26:22 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:36:13 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:36:14 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:36:18 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:36:18 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:37:10 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clear HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:51:00 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clear HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:51:21 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:51:22 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:51:33 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:51:36 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:51:53 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:51:54 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:07 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:08 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:16 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:18 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:24 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:26 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:31 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:33 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:44 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:50 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:53 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:52:53 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:55:02 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:55:04 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:55:10 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:55:10 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:55:15 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:55:15 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:03 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:06 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:22 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:29 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:29 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:32 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:33 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:40 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:40 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:52 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:56:52 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:57:02 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:57:03 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:57:25 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 08:57:25 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:07:22 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:07:22 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:07:55 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:07:55 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:07:58 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:07:58 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:05 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:05 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:10 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:10 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:15 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:27 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:27 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:37 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:08:37 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:10:05 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET / HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:10:30 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:10:30 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:11:31 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:11:31 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:11:34 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:11:34 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:11:48 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memched HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:12:00 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:12:01 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:12:09 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:12:09 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:14:18 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clear HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:14:38 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clear HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:14:47 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:14:48 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:14:57 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:14:57 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:16:08 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:16:08 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:16:15 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:16:15 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:16:21 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:16:21 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:16:26 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:16:27 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:28:48 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:28:48 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:29:26 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clear HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:29:43 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:29:43 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:29:59 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:29:59 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:30:13 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache' HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:30:18 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:30:19 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:30:25 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:30:25 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:33:07 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clear HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:33:20 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /faile HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:33:38 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:33:39 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:33:50 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:33:51 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:34:08 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:34:09 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:34:14 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:34:15 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:34:19 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:34:19 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:38:10 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:38:10 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:01 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:01 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:15 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:15 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:46 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:46 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:48 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:49 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:54 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:39:54 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:40:00 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:40:00 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:40:03 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:40:03 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:40:25 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:40:25 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:52:44 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clear HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:52:52 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:52:52 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:53:01 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:53:28 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:53:28 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:53:33 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:53:46 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:53:46 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:55:07 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:55:07 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:55:18 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:55:18 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:55:23 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:55:23 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:56:50 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:56:50 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:56:57 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:56:57 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:57:05 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:57:05 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:57:22 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:57:22 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:57:25 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 09:57:25 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 10:00:15 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 10:00:15 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /file HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 10:03:27 [error] 7#7: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clien HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 10:03:43 [error] 7#7: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /clear HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 10:03:52 [error] 7#7: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 10:03:53 [error] 7#7: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /redis HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 10:05:08 [error] 7#7: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
2022/09/22 10:05:08 [error] 7#7: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.21.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.21.0.1, server: den, request: "GET /memcache HTTP/2.0", upstream: "fastcgi://172.21.0.5:9000", host: "localhost"
